### PR TITLE
get data from noaa api by list of stations or particular station id

### PR DESCRIPTION
**Pull request by vera37**

### CODE REVIEW

File: server/serviceAccountKey.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (faisal63) [inline server/serviceAccountKey.json]: Please remove this file from git.

File: server/configKeys.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
```
Comment (faisal63) [inline server/configKeys.js]: Please remove these files from git

File: server/routes/noaaRoutes.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+var request = require('request');
+var express = require('express');
+var router = express.Router();
+
+
+router.get('/', (req, res) => {
+  var condition = '&datatypeid=TOBS&startdate=2010-05-01&enddate=2010-05-01&units=metric';
```
Comment (faisal63) [inline server/routes/noaaRoutes.js]: startdate and enddate shouldn't be fixed. Make these query params

**Comment by vera37** [comment]
I updated the request like this, if the user only pass a stationid, general info about the station will be returned, if a specific date is added, the data on that date will be returned.

**Comment by vera37** [comment]
I just added start date and end date. so the user could call the station data either based on date=xxx, or startdate=xxx&enddate=xxx.

**Pull request merged**